src/network/networkd-link.c

```diff
@@ -1996,6 +2006,12 @@ static int link_joined(Link *link) {
                         log_link_error_errno(link, r, "Could not set bridge message: %m");
         }
 
+        if (link->network->bridge || NETDEV_KIND_BRIDGE == netdev_kind_from_string(link->kind)) {
```

Comment:
nitpick: i'd just have done `streq(link->kind, "bridge")` bridge here, instead of involving `netdev_kind_from_string()`, but I figure that doesn't matter...